### PR TITLE
Preview Mode Should Not Cache

### PR DESCRIPTION
**Pull request by Timer**
This fixes a bug where preview mode would respond with server-side caching headers.

### CODE REVIEW

File: packages/next/build/webpack/loaders/next-serverless-loader.ts [hunk 1]
```diff
@@ -320,6 +320,9 @@ const nextServerlessLoader: loader.Loader = function() {
 
         const isFallback = parsedUrl.query.__nextFallback
 
+        const previewData = tryGetPreviewData(req, res, options.previewProps)
```
Comment (ijjk) [inline packages/next/build/webpack/loaders/next-serverless-loader.ts]: Looks like we need to add the import for this function

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| buildDuration | 10.3s | 10.3s | ⚠️ +21ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +1.94 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..b61f.js gzip | 12.3 kB | 12.3 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.22 kB | 9.22 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| index.html gzip | 923 B | 923 B | ✓ |
| link.html gzip | 933 B | 933 B | ✓ |
| withRouter.html gzip | 921 B | 921 B | ✓ |
| Overall change | 2.78 kB | 2.78 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| buildDuration | 11.1s | 10.5s | -579ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +1.94 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..b61f.js gzip | 12.3 kB | 12.3 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.22 kB | 9.22 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _error.js gzip | 289 kB | 289 kB | -47 B |
| 404.html gzip | 1.33 kB | 1.33 kB | ✓ |
| hooks.html gzip | 963 B | 963 B | ✓ |
| index.js gzip | 289 kB | 290 kB | ⚠️ +859 B |
| link.js gzip | 297 kB | 298 kB | ⚠️ +213 B |
| routerDirect.js gzip | 297 kB | 297 kB | ⚠️ +42 B |
| withRouter.js gzip | 297 kB | 296 kB | -244 B |
| Overall change | 1.47 MB | 1.47 MB | ⚠️ +823 B |

</details>

</details>


Commit: 7070d39c1b1103f5009e09b4d5390b2c8f2802fd

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| buildDuration | 10.7s | 10.9s | ⚠️ +239ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +1.94 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..b61f.js gzip | 12.3 kB | 12.3 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.22 kB | 9.22 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| index.html gzip | 923 B | 923 B | ✓ |
| link.html gzip | 933 B | 933 B | ✓ |
| withRouter.html gzip | 921 B | 921 B | ✓ |
| Overall change | 2.78 kB | 2.78 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| buildDuration | 11.8s | 12s | ⚠️ +154ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +1.94 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..b61f.js gzip | 12.3 kB | 12.3 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.22 kB | 9.22 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _error.js gzip | 289 kB | 289 kB | -80 B |
| 404.html gzip | 1.33 kB | 1.33 kB | ✓ |
| hooks.html gzip | 963 B | 963 B | ✓ |
| index.js gzip | 290 kB | 289 kB | -734 B |
| link.js gzip | 298 kB | 298 kB | -526 B |
| routerDirect.js gzip | 296 kB | 296 kB | ⚠️ +58 B |
| withRouter.js gzip | 297 kB | 296 kB | -161 B |
| Overall change | 1.47 MB | 1.47 MB | -1.44 kB |

</details>

</details>


Commit: 53938d5e1e85ad6e451f81e079af5c545d4f2c39

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| buildDuration | 9.2s | 9.6s | ⚠️ +410ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +2.28 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..b61f.js gzip | 12.3 kB | 12.3 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.22 kB | 9.22 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| index.html gzip | 923 B | 923 B | ✓ |
| link.html gzip | 933 B | 933 B | ✓ |
| withRouter.html gzip | 921 B | 921 B | ✓ |
| Overall change | 2.78 kB | 2.78 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| buildDuration | 10.6s | 10.7s | ⚠️ +42ms |
| nodeModulesSize | 54.5 MB | 54.5 MB | ⚠️ +2.28 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.05 kB | 5.05 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..b61f.js gzip | 12.3 kB | 12.3 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.14 kB | 4.14 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 9.22 kB | 9.22 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 53.2 kB | 53.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.js gzip | 1.06 kB | 1.06 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.9 kB | 1.9 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _app.module.js gzip | 577 B | 577 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.26 kB | 5.26 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js hotfix/preview-is-private | Change |
| - | - | - | - |
| _error.js gzip | 289 kB | 290 kB | ⚠️ +623 B |
| 404.html gzip | 1.33 kB | 1.33 kB | ✓ |
| hooks.html gzip | 963 B | 963 B | ✓ |
| index.js gzip | 289 kB | 290 kB | ⚠️ +819 B |
| link.js gzip | 298 kB | 298 kB | ⚠️ +82 B |
| routerDirect.js gzip | 297 kB | 296 kB | -335 B |
| withRouter.js gzip | 296 kB | 296 kB | ⚠️ +59 B |
| Overall change | 1.47 MB | 1.47 MB | ⚠️ +1.25 kB |

</details>

</details>


Commit: 5e80ddbc5115fdf0c4ec3012407c153efa2ea4b0

**Review by timneutkens — APPROVED** [review]

**Pull request merged**